### DISCUSSION TITLE
No error messages

### DISCUSSION THREAD

**Discussion by pharreX19** [vazco/form-builder-documentation#22] [Q&A]
Hello, Im using [vazco form builder](https://www.vazco.eu/form-builder), its working fine until recently I came to know that errors like "minNotGreaterThanMax" and other error messages are not showing up, I might have messed up something  when trying to fix another issues, does anyone know what i might have done wrong? any help is highly appreciated. Thanks

**Comment by caluskrzysztof** [comment]
Hello @pharreX19 could you provide an example? Based on the above description, not knowing the exact implementation we aren't able to reproduce it.

**Comment by radekmie** [comment]
To confirm what @caluskrzysztof said - I assume you meant a situation, when the form definition is incorrect, e.g., the "min length" of a text field is greater than the "max length" of it. In such case, a "Your model contains errors" error appears on the screen, and your modifications led to a situation where it's no longer appearing, correct?

If so, please check the `ErrorBoundary` component first and see, if the `modelErrors` array is present then. We can help you from there.

**Comment by pharreX19** [comment]
@radekmie thats exactly what I am referring to, the problem is I can clearly see the error in validation's to validator method. But within the error boundary its null even though FormEdit is wrapped within the ErrorBoundary. 
<img width="1437" alt="Screenshot 2023-02-19 at 10 36 09" src="https://user-images.githubusercontent.com/37008843/219928922-9728233f-3521-447a-a822-e8f52907494b.png">
![Screenshot 2023-02-19 at 10 39 55](https://user-images.githubusercontent.com/37008843/219929220-178dc464-fa70-469b-9334-9d79cf4f7b75.png)

**Comment by caluskrzysztof** [comment]
@pharreX19 Before we go any further, let's do one more thing and that is to verify that the Form Builder code you are using is up to date or does not have any major changes to the structure that may cause this behavior.
